CONTRIBUTING.md

```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
 
+Install Xcode and Homebrew Mac. Once that is done: 
```

Comment:
It seems that a gcc compiler is needed for Homebrew and pip, and xcode is one way to provide that but not the only way. What do you suggest I include? Just Homebrew and the landing page?